README.md

```diff
@@ -31,8 +31,8 @@ sudo apt install python3 python3-pip python3-venv build-essential libsnappy-dev
 Clone the repository and run the following commands from the root directory:
 
 ```
-python3 -m virtualenv venv
-source venv/bin/activate
+python3 -m virtualenv .venv
+source .venv/bin/activate
```

Comment:
Could you do the opposite fix and just rename .venv to venv in the .gitignore file?